manuscript/manuscript.Rmd

```diff
@@ -345,6 +349,15 @@ top_panel / bottom_panel
 ```
 
 
+
+This approach is not as different from conventional strategies at it may seem.
+In a conventional approach, ecological models are first estimated from empirical data, (stock assessments in the fisheries case).
+Quotas can then be set based directly on these model estimates, or by comparing alternative candidate "harvest control rules" (the policy function) against model-based simulations of stock dynamics.
+This latter approach, known in fisheries as Management Strategy Evaluation [MSE; @Punt2016] is already very closely analogous to the RL process.
+Instead of researchers evaluating a handful of control rules, the RL agent takes on the task of proposing and evaluating the millions of possible control rules represented by its neural network. 
```

Comment:
```suggestion
Instead of researchers evaluating a handful of control rules, the RL agent proposes and evaluates a plethora of possible control rules autonomously. 
```
It's hard for me to fit in neural networks in this sentence in a way that gibes with value-based methods for which the NN does not represent the policy. Besides, I think mentioning NNs might be distracting here to people that aren't familiar with NNs. The main point, I think you are trying to convey, is that RL tries a lot of policies/control rules and sees what works best.